src/components/AppNavigation/AppNavigation.vue

```diff
@@ -84,7 +85,20 @@ export default {
 	flex-direction: column;
 	flex-grow: 0;
 	flex-shrink: 0;
-
+}
+//list of navigation items
+ul {
+	position: relative;
+	height: 100%;
+	width: inherit;
+	overflow-x: hidden;
+	overflow-y: auto;
+	box-sizing: border-box;
+	display: flex;
+	flex-direction: column;
+	// Gives some room to the actions and popover
+	// menus
+	padding-right: 6px;
```

Comment:
I think this needs to be fixed in the actions directly :)
https://github.com/nextcloud/nextcloud-vue/blob/master/src/components/Actions/Actions.vue#L267

Maybe we should only adjust its position on align="center" :)

Reply:
Ah no, it actually help things. Without it it would be broken :)
https://github.com/nextcloud/nextcloud-vue/pull/474 works if the menu is not blocked by another div. The margin 10px is atually causing a bad shifting. We should maybe revert it to 0 and only add a margin-bottom of 10px.